src/platform-includes/performance/opentelemetry-setup/node.mdx

```diff
@@ -17,25 +25,50 @@ const {
 const {
   OTLPTraceExporter,
 } = require("@opentelemetry/exporter-trace-otlp-grpc");
+const {
+  AsyncLocalStorageContextManager,
+} = require("@opentelemetry/context-async-hooks");
+
+function setupSentry() {
+  setupGlobalHub();
+
+  // Make sure to call `Sentry.init` BEFORE initializing the OpenTelemetry SDK
+  Sentry.init({
+    dsn: "___PUBLIC_DSN___",
+    tracesSampleRate: 1.0,
+    // set the instrumenter to use OpenTelemetry instead of Sentry
+    instrumenter: "otel",
+    // ...
+  });
+
+  const client = getCurrentHub().getClient();
+  setupEventContextTrace(client);
 
-// Make sure to call `Sentry.init` BEFORE initializing the OpenTelemetry SDK
-Sentry.init({
-  dsn: "___PUBLIC_DSN___",
-  tracesSampleRate: 1.0,
-  // set the instrumenter to use OpenTelemetry instead of Sentry
-  instrumenter: "otel",
-  // ...
-});
-
-const sdk = new opentelemetry.NodeSDK({
-  // Existing config
-  traceExporter: new OTLPTraceExporter(),
-  instrumentations: [getNodeAutoInstrumentations()],
-
-  // Sentry config
-  spanProcessor: new SentrySpanProcessor(),
-  textMapPropagator: new SentryPropagator(),
```

Comment:
I wonder if we should get rid of the old approach entirely. There are still some people using it. Maybe we add a "DEPRECATED" section below with this code?

Reply:
Hmm, but I guess for people already using the old approach there isn't much need to go to the docs and find the same thing again? 😅 WDYT?

Reply:
yeah that's fair. We can also just link to the npm readme as a note then.

Reply:
Wait, does the npm readme have the old instructions? If so, I think linking to it would be a good solution.

Reply:
I added a section below (a note) pointing to the old readme, WDYT?